cpp/arcticdb/util/global_lifetimes.cpp

```diff
@@ -39,16 +44,23 @@ void ModuleData::destroy_instance() {
 
 void ModuleData::init() {
     ModuleData::instance_ = std::make_shared<ModuleData>();
+
+#if defined(_MSC_VER) && defined(_DEBUG)
+    if (auto config = ConfigsMap::instance()->get_int("CrtDbgDialog"); !config || config.value() == 0) {
+        log::root().info("_CrtSetReportMode(..., _CRTDBG_MODE_DEBUG)");
+        _CrtSetReportMode(_CRT_ERROR, _CRTDBG_MODE_DEBUG);
+        _CrtSetReportMode(_CRT_ASSERT, _CRTDBG_MODE_DEBUG);
+    }
+#endif
 }
 
 std::shared_ptr<ModuleData> ModuleData::instance_;
 std::once_flag ModuleData::init_flag_;
 
 void shutdown_globals() {
     async::TaskScheduler::destroy_instance();
-    storage::s3::S3ApiInstance::destroy_instance();
```

Comment:
Why don't we need this anymore?

Reply:
This is also on line 27/32, so we were destroying it twice.

Reply:
Ahah! Mystery solved